packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -457,11 +459,25 @@ export class CallContext {
   }
 
   /* @conditional-compile-remove(together-mode) */
-  public setTogetherModeVideoStream(callId: string, addedStream: TogetherModeVideoStream[]): void {
+  public setTogetherModeVideoStream(callId: string, addedStreams: TogetherModeVideoStream[]): void {
```

Comment:
Again, can we have this an an object for the TogetherModeVideoStream instead of an array?

Reply:
This method is internal and the TogetherModeVideoStream array is from SDK.